docker-compose.production.yml

```diff
@@ -8,17 +8,22 @@ version: "3.1"
 services:
   web:
     restart: always
-    hostname: ${HOSTNAME:-$HOST}
+    deploy:
+      replicas: 2
+    hostname: "$HOSTNAME"
     environment:
       - GUNICORN_OPTS= --workers 50 --timeout 300 --max-requests 500
       - OL_CONFIG=/olsystem/etc/openlibrary.yml
       - PYENV_VERSION=3.8.6
     volumes:
-      - ../olsystem:/olsystem
       - ../booklending_utils:/booklending_utils
+      - ../olsystem:/olsystem
+      - ../olsystem/etc/ia.ini:/openlibrary/.config/ia.ini
   covers:
     restart: always
-    hostname: ${HOSTNAME:-$HOST}
+    deploy:
+      replicas: 1
```

Comment:
Let's not add these just yet; switching to docker swarm/stack would be a mess of pitfalls right now. We don't want to use them because it will misleading make it seem like we are using swarm/stack when we aren't. But someday :)